[ayucloud.services]
auth_path  user
auto_invite  F
banned_domains  xxxhi.cc  snapmail.cc
buy  pass
email  fwcjcivsv118@jkd.tmmad.com
last_checkin  2025-05-03 15:35:24.535236+08:00
name  AyucLouD-Services
node_n  99
sub_info  0B  273M  2025-05-02 22:13:30+08:00  (剩余 273M -2 days, 21:57:46.472255)
sub_url  https://network.ayucloud.nl/link/76rRwPr64FeltmZP?clash=1
time  2025-05-01 22:13:31.479370+08:00
type  sspanel
保存base64/clash订阅失败  'NoneType' object is not iterable
尝试签到失败  HTTPSConnectionPool(host='ayucloud.services', port=443): Max retries exceeded with url: /user/login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f507d665f00>, 'Connection to ayucloud.services timed out. (connect timeout=30)'))
更新旧订阅失败  HTTPSConnectionPool(host='ayucloud.services', port=443): Max retries exceeded with url: /user/login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f507d657520>, 'Connection to ayucloud.services timed out. (connect timeout=30)'))

[qlgq.top]

[cyanmori.com]

[vipxgyun.com]
